x/ccv/provider/keeper/relay.go

```diff
@@ -390,6 +459,7 @@ func (k Keeper) HandleSlashPacket(ctx sdk.Context, chainID string, data ccv.Slas
 			sdk.NewAttribute(ccv.AttributeValSetUpdateID, strconv.Itoa(int(data.ValsetUpdateId))),
 		),
 	)
+	k.Logger(ctx).Debug("handled slash packet", "chainID", chainID, "provider cons addr", providerConsAddr.String(), "infraction type", data.Infraction, "infraction height", infractionHeight)
```

Comment:
How about,
 after `k.stakingKeeper.Jail` log that it was jailed
 remove the else
 after `k.slashingKeeper.JailUntil` log that the jail time was extended